downstream/modules/platform/proc-edge-manager-build-bootc-image.adoc

```diff
@@ -0,0 +1,91 @@
+[id="edge-manager-build-bootc-image"]
+
+= Building the operating system image with _bootc_
+
+Build the operating system image with `bootc` that contains the {RedHatEdge} agent. 
```

Comment:
Is it the bootc that contains the agent or the operating system image? If it's the bootc, I suggest changing to include the definite article "the":

Build the operating system image with the `bootc` that contains the {RedHatEdge} agent.